library/modules/bigip_device_auth.py

```diff
@@ -320,11 +342,15 @@ def servers(self):
                         "An 'address' field must be provided when specifying separate fields to the 'servers' parameter."
                     )
                 address = server.get('address')
-                port = server.get('port', 49163)
```

Comment:
while i understand why you did this (i.e no defaults), changing the 'default' now will cause issues for users that were relying on this in their plays. avoid doing such changes without adding deprecation warnings

Reply:
if this was needed cause the module was breaking then this kind of user breaking change would be acceptable

Reply:
it actually doesn't break user experience because the user doesn't want the port to be specified, and BIG-IP documentation does't recommend it. So the user will end up in a positive situation according to my read of the issue.